addons/l10n_dk_oioubl/tests/test_files/from_odoo/oioubl_out_refund_partner_dk.xml

```diff
@@ -142,22 +142,6 @@
       </cac:Contact>
     </cac:Party>
   </cac:AccountingCustomerParty>
-  <cac:Delivery>
-    <cac:DeliveryLocation>
-      <cac:Address>
-        <cbc:AddressFormatCode listAgencyID="320" listID="urn:oioubl:codelist:addressformatcode-1.1">
-          StructuredDK</cbc:AddressFormatCode>
-        <cbc:StreetName>Paradisæblevej,</cbc:StreetName>
-        <cbc:BuildingNumber>11</cbc:BuildingNumber>
-        <cbc:CityName>Aalborg</cbc:CityName>
-        <cbc:PostalZone>9430</cbc:PostalZone>
-        <cac:Country>
-          <cbc:IdentificationCode>DK</cbc:IdentificationCode>
-          <cbc:Name>Denmark</cbc:Name>
-        </cac:Country>
-      </cac:Address>
-    </cac:DeliveryLocation>
-  </cac:Delivery>
```

Comment:
interesting, FYI @Levizar
we were adding the Delivery node in the UBL 2.0 template on the document level while it's only present in UBL 2.1 (see http://www.datypic.com/sc/ubl20/e-ns14_CreditNote.html)

Reply:
@JulienVR Thx for the insight. It wasn't blocking for the schematron nor the XSD soooo :man_shrugging: